packages/kotti-ui/source/kotti-popover/components/IconTextItem.vue

```diff
@@ -3,12 +3,14 @@
 		class="kt-popover-options-item"
 		:class="{
 			'kt-popover-options-item--is-disabled': isDisabled,
+			'kt-popover-options-item--is-selected': isSelected,
 		}"
 		:data-test="dataTest"
+		:tabindex="isDisabled ? -1 : 0"
```

Comment:
Does `-1` work? Otherwise, I'd prefer `undefined`

Reply:
https://www.techonthenet.com/html/elements/global_attributes.php#:~:text=tabindex,reachable%20through%20sequential%20keyboard%20navigation

Reply:
if I set it to be `undefined`, the element is not focusable, then the `event.relatedTarget` that's captured by the `blur` `event` from the `input` element (see https://github.com/3YOURMIND/kotti/pull/450/files#diff-e7f6fccf48a52fe6a4fffaa684158edc9d0c5a85bb8c153a41f959505889ebfaR182) is the component that is the parent of the tippy content so the function  returns `false` and so we close the dropdown

resolving, will leave as is.